pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -7,6 +7,7 @@ buildLinux (args // {
   src = fetchFromGitHub {
     owner = "koverstreet";
     repo = "bcachefs";
+    # commit does not exist on any branch on the target repository
```

Comment:
I found this on `master`: https://github.com/koverstreet/bcachefs/commits/master?after=6a505b63ed3003faf5000f19fd08bbd477d93fbc+139&branch=master

Reply:
I did not want to change anything here because I was to lazy to regenerate the deps.nix.

Reply:
No, I mean, I found this exactly commit in `master`.

Reply:
Just look at the link I posted. It is exactly this commit and this is the show of the `master` branch (just after the commit you asked).

Reply:
Maybe your script is bugged :thinking: ?

Reply:
NVM GitHub seems to be even more dumb than I thought (if I change `master` to any other valid branch it shows the same history).